filebeat/_meta/common.reference.inputs.yml

```diff
@@ -231,8 +231,8 @@ filebeat.inputs:
   # Network type to be used for redis connection. Default: tcp
   #network: tcp
 
-  # Max number of concurrent connections. Default: 10
-  #maxconn: 10
+  # Max number of concurrent connections. By default it is not limited.
+  #maxconn: 0
```

Comment:
It looks like "maxconn" doesn't exist in the configuration, was this supposed to be removed?

Also make the documentation more explicit (after moving it below), e.g. "Max number of concurrent connections, or 0 for no limit. Default: 0"

Reply:
:man_facepalming: